### PR TITLE
nuget CPM support

### PR DESCRIPTION
**Pull request by WeihanLi**
## Proposed Changes

Use the nuget CPM feature to manage the nuget package versions centrally

https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management

## Types of Changes

What types of changes does your code introduce to this project?
_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes issue #NNNN)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause an observable behavior change in existing systems)
- [ ] Documentation improvements (corrections, new content, etc)
- [ ] Cosmetic change (whitespace, formatting, etc)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating
the PR. If you're unsure about any of them, don't hesitate to ask on the
mailing list. We're here to help! This is simply a reminder of what we are
going to look for before merging your code._

- [x] I have read the `CONTRIBUTING.md` document
- [x] I have signed the CA (see https://cla.pivotal.io/sign/rabbitmq)
- [x] All tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in related repositories

### CODE REVIEW

**Comment by WeihanLi** [comment]
submodule project needs to be updated also
https://github.com/rabbitmq/toxiproxy-netcore/pull/1

File: projects/RabbitMQ.Client.OpenTelemetry/RabbitMQ.Client.OpenTelemetry.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (lukebakken) [inline projects/RabbitMQ.Client.OpenTelemetry/RabbitMQ.Client.OpenTelemetry.csproj:1]: @WeihanLi whatever text editor you are using is removing the UTF-8 BOM from these files, and creating an unnecessary change. Please revert.

File: projects/Test/Common/Common.csproj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (lukebakken) [inline projects/Test/Common/Common.csproj:1]: @WeihanLi whatever text editor you are using is removing the UTF-8 BOM from these files, and creating an unnecessary change. Please revert.

File: projects/Test/SequentialIntegration/SequentialIntegration.csproj [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (lukebakken) [inline projects/Test/SequentialIntegration/SequentialIntegration.csproj:1]: @WeihanLi whatever text editor you are using is removing the UTF-8 BOM from these files, and creating an unnecessary change. Please revert.

**Review by lukebakken — CHANGES_REQUESTED** [review]

File: projects/RabbitMQ.Client.OpenTelemetry/RabbitMQ.Client.OpenTelemetry.csproj (hunk 1 above)
Comment (WeihanLi) [reply projects/RabbitMQ.Client.OpenTelemetry/RabbitMQ.Client.OpenTelemetry.csproj]: @lukebakken  thanks very much for help, I did not notice this before

**Review by lukebakken — APPROVED** [review]
Thank you!

**Comment by WeihanLi** [comment]
thanks for help
